### PR TITLE
Fix typo of InsertDataToConfigMap func description

### PR DESCRIPTION
**Pull request by carlory**
**Please provide a description of this PR:**

https://github.com/istio/istio/pull/43870 changed it but forget to update the comment.

### CODE REVIEW

File: security/pkg/k8s/configutil.go [hunk 1]
```diff
@@ -27,8 +27,7 @@ import (
 )
 
 // InsertDataToConfigMap inserts a data to a configmap in a namespace.
-// client: the k8s client interface.
-// lister: the configmap lister.
+// client: the wrapped k8s client interface.
 // meta: the metadata of configmap.
 // caBundle: ca cert data bytes.
```
[the remark this answers is not in the document]
Comment (howardjohn) [reply security/pkg/k8s/configutil.go:32]: yeah that sgtm

**Review by lei-tang — APPROVED** [review]

**Review by howardjohn — COMMENTED** [review]
thanks!

**Pull request merged**